cmd/influx/replication.go

```diff
@@ -85,6 +85,12 @@ func newReplicationCreateCmd() cli.Command {
 				Usage:       "Do not drop data when a non-retryable error is encountered",
 				Destination: &params.NoDropNonRetryableData,
 			},
+			&cli.Int64Flag{
+				Name:        "max-age",
+				Usage:       "Specify a maximum age (in seconds) for replications data before it is dropped, or 0 for infinite",
+				Value:       604800, // source: https://github.com/influxdata/openapi/blob/6ea7df4daa5735a063be3db60d0165b34b26c096/src/oss/schemas/ReplicationCreationRequest.yml#L27
```

Comment:
This comment seems likely to drift. Maybe just give the file and not the line number?